src/ontology/mondo-ingest.Makefile

```diff
@@ -419,6 +423,8 @@ tmp/merged.db: tmp/merged.owl
 	@rm -f tmp/merged-relation-graph.tsv.gz
 
 ../mappings/mondo-sources-all-lexical.sssom.tsv: $(SCRIPTSDIR)/match-mondo-sources-all-lexical.py tmp/merged.db
+	rm -f ../mappings/mondo-sources-all-lexical.sssom.tsv
+	rm -f ../mappings/mondo-sources-all-lexical-2.sssom.tsv
```

Comment:
what's the difference between the `lexical` and `lexical-2`?

Reply:
`-2` is a filtered version of the other. It has `subject_id` starts with `MONDO:%`.